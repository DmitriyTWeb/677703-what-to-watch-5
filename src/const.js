export const RatingDesc = {
  BAD: `Bad`,
  NORMAL: `Normal`,
  GOOD: `Good`,
  VERY_GOOD: `Very good`,
  AWESOME: `Awesome`,
};

export const DEFAULT_RENDERED_FILMS_COUNT = 8;
export const PLAYING_TIMEOUT = 1000;

export const AuthorizationStatus = {
  AUTH: `AUTH`,
  NO_AUTH: `NO_AUTH`,
};

export const AppRoute = {
  LOGIN: `/login`,
  ROOT: `/`,
  MYLIST: `/mylist`,
  FILMS_ID: `/films/:id`,
  PLAYER_ID: `/player/:id`,
  REVIEW_ID: `/films/:id/review`,
  ERROR: `/error`,
};

export const APIRoute = {
  FILMS: `/films`,
  LOGIN: `/login`,
  LOGOUT: `/logout`,
  PROMO: `/films/promo`,
};
