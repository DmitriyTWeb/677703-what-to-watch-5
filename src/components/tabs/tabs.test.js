import React from "react";
import renderer from "react-test-renderer";
import Tabs from "./tabs";
import {mockTab} from "../../const.js";

const tabs = [mockTab, mockTab, mockTab];

jest.mock(`../tab/tab.jsx`, () => () => `Tab`);

it(`Should Tabs render correctly`, () => {
  const tree = renderer.create(
      <Tabs
        tabs={tabs}
        onTabClick={() => {}}
        activeTabId={tabs[0].id}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
