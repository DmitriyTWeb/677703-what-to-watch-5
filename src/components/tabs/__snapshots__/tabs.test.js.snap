// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Tabs render correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    Tab
    Tab
    Tab
  </ul>
</nav>
`;
